source/css/sass/mixins/_typography.scss

```diff
@@ -3,6 +3,7 @@
 @import "../variables/color";
 @import "../variables/font";
 @import "spacing";
+@import "utilities";
```

Comment:
Not a fan of `utilities` as it's too generic, but not sure where else to put it. 😐